src/main/kotlin/com/klaxit/hiddensecrets/HiddenSecretsPlugin.kt

```diff
@@ -99,10 +99,10 @@ open class HiddenSecretsPlugin : Plugin<Project> {
         @Input
         fun getKeyNameParam(): String {
             val chars = ('a'..'z') + ('A'..'Z')
-            // Default random key name
+            //  Default random key name
```

Comment:
Double space

Reply:
> Hello and thanks for the PR. I totally agree with these changes.
> 
> Can you just fix the 2 errors please ?
> 
> Thanks

Oh, absolutely! My confidence in find & Replace was a little high apparently :-)